src/main/java/com/avispl/symphony/dal/communicator/aver/ptz/UDPCommunicator.java

```diff
@@ -29,15 +29,15 @@
  */
 public class UDPCommunicator extends BaseDevice implements Communicator {
```

Comment:
Please remove the unused import 
`import java.util.Iterator`

Reply:
I will fix it